src/checks/addMilestoneLabelsToPRs.ts

```diff
@@ -0,0 +1,56 @@
+import { WebhookPayloadPullRequest } from "@octokit/webhooks"
+import { Octokit } from "@octokit/rest"
+import { Logger } from "@azure/functions"
+import { getRelatedIssues } from "../pr_meta/getRelatedIssues"
+
+/**
+ * Keep track of the milstone
+ */
+export const addMilestoneLabelsToPRs = async (api: Octokit, payload: WebhookPayloadPullRequest, logger: Logger) => {
```

Comment:
this gets run on each PR periodically, I take it?

Reply:
Any of these trigger it:

![Screen Shot 2020-07-15 at 12 12 30 PM](https://user-images.githubusercontent.com/49038/87568969-7c3c2f80-c694-11ea-9934-3e19ebe6fb36.png)

I will probably make a daily PR run